wandb/apis/public.py

```diff
@@ -448,6 +448,10 @@ def create_report(
         ).save()
 
     def create_project(self, name: str, entity: str):
+        if name == "model-registry2":
+            raise wandb.Error(
+                "Cannot create proejct with protected name, model-registry"
```

Comment:
proejct -> project

Reply:
It's also lying.  How about f"{name} is reserved, please choose a different project name"

Reply:
Also do you really mean `model-registry2`?

Reply:
No, sorry - typo. It was meant to be `model-registry`